packages/report-ui/src/components/restqa/test-report/charts/Features.vue

```diff
@@ -61,6 +71,18 @@ export default {
     nbFeatures() {
       return this.data.features.length;
     },
+    statusImageUrl() {
+      return `images/reportStatus/${
+        this.$store.getters.result.success
+          ? "statusSuccess.png"
+          : "statusFailed.png"
+      }`;
+    },
+    statusDesc() {
+      return this.$store.getters.result.success
```

Comment:
please use the `this.data` instead of fetching data directly from the store.
Then the component is decoupled from the data source.

Reply:
Updated.